plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -61,6 +64,45 @@ public override IEnumerable<FileModel> Prebuild(ImmutableList<FileModel> models,
                         collection.AddRange(result.Models);
                     }
                 }
+
+                var duplicatePathes = filePathsDict.Where(x => x.Value.Item1).Select(x => x.Key).ToList();
```

Comment:
>var duplicatePathes = filePathsDict.Where(x => x.Value.Item1).Select(x => x.Key).ToList(); [](start = 16, length = 90)

Call `Select` inside `foreach models` is too time consuming. Can we consume filePathsDict outside of foreach?